doc/nrf/external_comp/dult.rst

```diff
@@ -47,6 +47,7 @@ The DULT integration in the |NCS| consists of the following steps:
 #. :ref:`Managing the identification process <ug_dult_identifier>`
 #. :ref:`Using the sound callbacks and managing the sound state <ug_dult_sound>`
 #. :ref:`Managing the battery information <ug_dult_battery>`
+#. :ref:`Interacting with the motion detector <ug_dult_motion_detector>`
```

Comment:
The step list is not in line with the step numbers below. Either move this item above `Managing the battery information` or move the section content to the correct place